components/commons/overlay-coretokens.css

```diff
@@ -29,26 +28,22 @@ governing permissions and limitations under the License.
 %spectrum-overlay--open {
   visibility: visible;
   opacity: 1;
-  transition-delay: 0ms;
+  transition-delay: var(--mod-animation-duration-0, var(--spectrum-animation-duration-0));
```

Comment:
We'll want to assign this to a component-specific variable before modding it. Just like you did at the top of this file you'll want to do something like 
`--spectrum-overlay-animation-duration-0: var(--spectrum-animation-duration-0)` and then using that variable you create in this spot